### PR TITLE
Add docker env for local development

### PR DESCRIPTION
**Pull request by lociii**
To simplify local development and reduce the need to have virtualenv and other libs installed, this PR adds a simple docker container to run tests in.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+WORKDIR /app
+
+ADD runtests/requirements.txt /app/
+
+USER app
+
+ENV PATH /home/app/venv/bin:${PATH}
+
+RUN python3 -m venv ~/venv && \
```
Comment (auvipy) [inline Dockerfile:33]: do we really need to use virtualenv with docker?

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+version: "3.5"
```
Comment (auvipy) [inline docker-compose.yml:1]: is it python version?

**Review by auvipy — CHANGES_REQUESTED** [review]
thanks for your contribution. waiting for the review comments

File: docker-compose.yml (hunk 2 above)
Comment (regiohelden-dev) [reply docker-compose.yml:1]: It's the Compose file format version, see https://docs.docker.com/compose/compose-file/

Comment (lociii) [reply docker-compose.yml:1]: It's the Compose file format version, see https://docs.docker.com/compose/compose-file/

File: Dockerfile (hunk 1 above)
Comment (lociii) [reply Dockerfile:33]: I prefer to have my Docker setup as close as possible to my production setups which use virtualenvs.
But can be removed if you don't like it.

**Review by auvipy — APPROVED** [review]

**Comment by auvipy** [comment]
thanks!

File: Dockerfile (hunk 1 above)
Comment (auvipy) [reply Dockerfile:33]: wondering why didn't we consider using FROM python:3.8-slim-buster?

**Pull request merged**